README.md

```diff
@@ -2,6 +2,20 @@
 
 Icons, fonts, colors, components and all user interface items for OSRD project.
 
+# Build Instructions
+
+1. Go to the project's `root` directory in your terminal
+2. Run `sudo chown -R $(whoami) ~/.npm`
+3. Enter your computer's password
+4. Run `rm -rf node_modules package-lock.json`
+5. Run `npm cache clean --force`
+6. Run `npm install @nx/nx-darwin-x64`
+7. Run `npm install @rollup/rollup-darwin-x64`
+8. Run `npm run clean`
+9. Run `npm run build`
+10. Run `npm run test`
```

Comment:
Running the tests isn't strictly required to build the project. Maybe could be mentioned in another sentence?

Reply:
Good point. Thanks :)